packages/financial-templates-lib/src/price-feed/DefaultPriceFeedConfigs.js

```diff
@@ -951,6 +951,20 @@ const defaultConfigs = {
         invertPrice: true
       }
     }
+  },
+  USDDEXTF: {
+    type: "expression",
+    expression: "ETHDEXTF * USDETH",
+    lookback: 7200,
+    minTimeBetweenUpdates: 60,
+    twapLength: 3600,
```

Comment:
Also currently the UMIP specifies that the 5-min TWAP is only applied to the ETHDEXTF price and not the USDETH price. Specifying `twapLength` at this level applies it to both constituent price feeds. If following the specs of the UMIP, this should be moved into the `ETHDEXTF` customFeed below. 

If it seems like a better approach to apply the twap to both, the UMIP can be updated instead. What would you prefer?

Reply:
Ok to apply `twapLength` to both constituent price feeds